src/main/java/vmware/samples/contentlibrary/delete/ContentLibraryDelete.java

```diff
@@ -0,0 +1,162 @@
+package vmware.samples.contentlibrary.delete;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.commons.cli.Option;
+import org.apache.commons.lang.StringUtils;
+
+import com.vmware.content.LibraryModel;
+import com.vmware.content.LibraryTypes;
+import com.vmware.content.library.StorageBacking;
+import com.vmware.vim25.ManagedObjectReference;
+
+import vmware.samples.common.SamplesAbstractBase;
+import vmware.samples.common.vim.helpers.VimUtil;
+import vmware.samples.contentlibrary.client.ClsApiClient;
+
+public class ContentLibraryDelete extends SamplesAbstractBase {
```

Comment:
Why do we need a separate sample for delete ? Shouldn't this be in LibraryCrud ?

Reply:
LibraryCRUD is not the only sample creating Content Libraries. This sample was added to all the content libraries in the DC or one library by name.